eventuals/finally.h

```diff
@@ -45,15 +43,28 @@ struct _Finally final {
   };
 
   struct Composable final {
-    template <typename Arg>
-    using ValueFrom = expected<Arg, std::exception_ptr>;
+    template <typename...>
+    struct AddStopped {};
+
+    template <typename... Errors>
+    struct AddStopped<std::variant<Errors...>> {
+      using type = std::variant<Stopped, Errors...>;
+    };
+
+    template <typename Arg, typename Errors>
+    using ValueFrom = expected<
+        Arg,
+        typename AddStopped<typename TupleToVariant<Errors>::type>::type>;
 
     template <typename Arg, typename Errors>
     using ErrorsFrom = std::tuple<>;
 
-    template <typename Arg, typename K>
+    template <typename Arg, typename Errors, typename K>
     auto k(K k) && {
-      return Continuation<K, Arg>{std::move(k)};
+      return Continuation<
+          K,
+          typename AddStopped<typename TupleToVariant<Errors>::type>::type,
```

Comment:
This will have to be tweaked as per above. How about a `using Expected = ` type alias so we don't have to repeat the long `std::conditional_t` and then we can just have `using ValueFrom = Expected<Arg>;`.